documentation/asciidoc/computers/ai/getting-started.adoc

```diff
@@ -96,24 +97,20 @@ The `rpicam-apps` suite of camera applications implements a xref:camera_software
 
 The following demos use xref:camera_software.adoc#rpicam-hello[`rpicam-hello`], which by default displays a preview window. However, you can use other `rpicam-apps` instead, including xref:camera_software.adoc#rpicam-vid[`rpicam-vid`] and xref:camera_software.adoc#rpicam-still[`rpicam-still`]. You may need to add or modify some command line options to make the demo commands compatible with alternative applications.
 
-To begin, download the post-processing JSON files required for the demos. These files determine which post-processing stages to run and configure the behaviour of each stage. For example, you can enable, disable, strengthen, or weaken the strength of the temporal filtering in the object detection demos. Or you could enable or disable the output mask drawing in the segmentation demo.
-
-To download the entire collection of post-processing JSON files, clone the `rpicam-apps` repo. Run the following command to clone only the most recent commit from the repo, saving space:
+To begin, ensure you have the latest `rpicam-apps` software package installed with:
 
 [source,console]
 ----
-$ git clone --depth 1 https://github.com/raspberrypi/rpicam-apps.git ~/rpicam-apps
+$ sudo apt install rpicam-apps
```

Comment:
Should we suggest that users run an `sudo apt update` first?

Reply:
probably!

Reply:
```suggestion
$ sudo apt update && sudo apt install rpicam-apps
```
This should be enough, no?